bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -188,10 +202,35 @@ <h2>{{ ftl('home-mozilla-makes-privacy') }}</h2>
     </div>
   </div>
 
-  <section class="c-mission mzp-l-content">
+  <section class="c-mission">
+
+    <!-- Trustworthy AI -->
+    {% call split(
```

Comment:
**Question:** seeing as the Trustworthy AI and "What is Mozilla?" sections are new content, should we only show these splits if the page has them translated? Could we maybe fall back just to this for languages that don't yet have the new content translated?

![image](https://github.com/mozilla/bedrock/assets/400117/ca48165e-a1d0-4095-9e48-c55e0e40124f)

Reply:
Great idea! I added the `ftl_has_messages` helper for the Trustworthy AI section, however I've already set the fallbacks for the "What is Mozilla?" section to be the former strings you screenshotted above strings